internal/activity.go

```diff
@@ -353,6 +358,16 @@ func WithLocalActivityTask(
 	})
 }
 
+func NewExtraInformation() ActivityExtraInformation {
+	info := ActivityExtraInformation{}
+	return info
+}
+
+func (b ActivityExtraInformation) WithRetryDelay(delay time.Duration) ActivityExtraInformation {
```

Comment:
Would recommend just letting users set this as a field

Reply:
Yes please, we don't use the builder pattern in this SDK AFAIK.